### ISSUE TITLE
Net ties

### ISSUE THREAD

**Issue by ghost** [KiCad/Oddities.pretty#2]
Reviewing some old PRs, I came across this one proposing a symbol for net ties ...

https://github.com/KiCad/kicad-library/pull/457

This led me to search for net tie footprints, which I found here in this deprecated `Oddities.pretty` repo.

Q1. Should we retain the net tie footprints?  If so, which repo should they move to?

Q2. I don't believe there are any schematic symbols for net ties currently in `kicad-library`. Either via that PR above or otherwise, should we add symbols to match the footprints?

**Comment by evanshultz** [comment]
@hackscribble 
Thanks for bringing this up!

Tying nets together at a specific point on a board is a key part of any PCB design program, which I wrote up as an issue for the devs at https://bugs.launchpad.net/kicad/+bug/1571930. While I haven't seen any progress on a new/different solution, I strongly believe these footprints should be retained to allow net ties (as mentioned at https://github.com/KiCad/kicad-library/issues/1358#issuecomment-311216815).

I've already made matching schematic symbols that I will add to a PR this weekend.

The name `Oddities` is awful, though, and something needs to be done about that. I started writing down some ideas about footprint library naming at https://github.com/KiCad/Connectors_WAGO.pretty/issues/1, but I don't have a good idea where they should go. Even `Miscellaneous` is better than `Oddities`.
